manual/query_timestamps/README.md

```diff
@@ -116,6 +83,43 @@ When this happens, the built-in generators log a periodic warning message in the
 `com.datastax.driver.core.TimestampGenerator`. See their non-default constructors for ways to control the warning
 interval.
 
+
+#### Overriding the timestamp manually
+
+You can override the generator and provide and explicit timestamp for a given query: 
+
+```java
+Statement statement = new SimpleStatement(
+    "UPDATE users SET email = 'x@y.com' where id = 1");
+statement.setDefaultTimestamp(1234567890);
+session.execute(statement);
+```
+
+
+### Server-side generation
+
+This is the "legacy" behavior if you're connected to a Cassandra version
+that only supports protocol v2 or below. The server will assign a
+timestamp based on the time it received the query.
+
+This can be a problem when the order of the writes matter: with unlucky
+timing (different coordinators, network latency, etc.), two successive
+requests from the same client might be processed in a different order
+server-side, and end up with out-of-order timestamps. This is why client-side
+generation is the default starting with protocol v3. With older versions, the
+only way to get client timestamps is to add a `USING TIMESTAMP` clause to your
+CQL queries.
+
+Note that, although client-side timestamps are recommended when available, you can still opt for 
```

Comment:
It could be good to indicate when Server-side generation might be more viable than client side, i.e. if you have many more clients than C*, or where it is more difficult to ensure time is sync'd across clients vs. managing time on the C* nodes.